### ISSUE TITLE
Writing Documentation: Move things from the Wiki

### ISSUE THREAD

**Issue by donehere** [TYPO3-Documentation/TYPO3CMS-Guide-HowToDocument#43]
Most things can be merged, for some it might make sense to keep the Wiki page but add a link. 

* [x] https://wiki.typo3.org/ReST_Syntax (text is well written and some of the explanations shorter and more to the point, see if this can be merged in ..)
* [x]  https://wiki.typo3.org/Documentation_images
* [x] https://wiki.typo3.org/Editors_(reST)
* [x] https://wiki.typo3.org/Contributing_to_the_documentation (marked for deletion)
* ~~https://wiki.typo3.org/Alternative_Language~~ (concerns Wiki only)
* [x] https://wiki.typo3.org/DocTeam/Official_Documentation_Screenshots (added outdated tag)
* [x] https://wiki.typo3.org/Rendering_reST_on_Linux

categories:

* https://wiki.typo3.org/Category:DocTeam
* https://wiki.typo3.org/Category:About_documentation

other

* this https://wiki.typo3.org/Documentation_Process_-_Problems_and_Resolutions might rather be moved to the protocols in the forge wiki: https://forge.typo3.org/projects/team-docteam/wiki

## When moving please make sure to always

* add a link to new content
* always add the category Moved: `[[category: Moved]]`

**or better: create redirect on server and remove the page** (see discussion in #typo3-wiki channel on Slack)

see: https://wiki.typo3.org/WikiMigration

**Comment by josefglatz** [comment]
Even the UTF-8 converting thematic is important

**Comment by sypets** [comment]
@josefglatz 

> Even the "database UTF-8 converting" wiki page is important

Can you add a link? 

btw, more and more people are complaining about outdated information in the Wiki and that they often get search results with a Wiki page with outdated information as one of the top results. There is a growing number of people who consider current state of outdated information far more harmful than removing a few pages that are actually still useful.

**Comment by josefglatz** [comment]
@sypets Hi, shure, here's the link: https://wiki.typo3.org/UTF-8_support#Convert_an_already_existing_database_to_UTF-8

I know some upgrade projects in 2020 where it' still necessary to update a database (structure and/or content; fully/partially). Some information there might be outdated. I speak today with my colleague. Let's see if we can write some initial content

**Comment by sypets** [comment]
Ah, I vaguely remember doing that a few years ago. I found the information for that elsewhere, not in the TYPO3 wiki.

Question: is this TYPO3 specific? There is some TYPO3 specific information there but it seems to be outdated. Most of the rest is not TYPO3 specific. Unless someone is willing to update the page, I would remove it. Even if someone is willing, the difficulty is to make the information available for several versions and keep it maintained. 

There are a bunch of Wiki pages which are not really TYPO3 specific. While I think it is nice to have them the entire content of TYPO3 documentation (including docs.typo3.org + Wiki) is not maintainable currently. For docs.typo3.org, I think we are getting there, but for the Wiki this remains a problem. Currently there is not even an official maintainer for the Wiki.

So what I would suggest (for this specific topic):

* if the information is TYPO3 specific and important enough - add it to this guide
* if not, tag the page for deletion `{{delete|reason and optional link}}`

----

The general question of what should happen with the Wiki - we have been trying to clarify this for at least 2 years now. I made several attempts to start decision topics. Whenever you suggest to delete something, someone comes along and say, but we still need this ... 

If this gets dragged out any longer it will drive away the few remaining people who are still willing to do something.

Right now, the topic is in the hands of Susi.
